openapi-v1.yaml

```diff
@@ -6718,6 +6768,44 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /invitations/{namespace}/{group}/share/group:
+    parameters:
+      - name: namespace
+        in: path
+        description: namespace group is in (an organization name or user's username)
+        type: string
+        required: true
+      - name: group
+        in: path
+        description: name/uri of group that is url-encoded
+        type: string
+        required: true
+    post:
+      tags:
+        - invitation
+      description: Sends email to multiple recipients with sharing information regarding a group
+      operationId: shareGroupByInvite
+      parameters:
+        - name: group_email_invite
+          in: body
+          description: list of email/name recipients
+          schema:
+            $ref: "#/definitions/InvitationGroupShareEmail"
+          required: true
+      responses:
+        204:
+          description: Email sent successfully to user for email confirmation link
```

Comment:
Maybe rephrase to

Email sent successfully to user with an email confirmation link